lib/scrabble.rb

```diff
@@ -10,4 +11,25 @@ def point_values
       "Y"=>4, "Z"=>10
     }
   end
+
+  def score(word)
+    if word == nil
+      0
+    else
+      point_values.values_at(*word.upcase.split(//)).sum
+    end
+  end
+
+  def score_with_multipliers(words, loc, double = 1)
+    if words == nil
+        0
+    elsif words.length >= 7
+      values = point_values.values_at(*words.upcase.split(//))
```

Comment:
You seem to be reusing this line of code quite a bit - I think it would make a great helper method!